src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    public static final int BOUND = 100;
+
+    private int radius;
+
+    public Circle() {
+    }
+
+    public Circle(String color, int radius) {
+        super.setColor(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void setRandomProperties() {
+        super.setRandomProperties();
+        this.radius = random.nextInt(BOUND);
+    }

```

Comment:
do we need this method? (we can set fields through constructor)
Please, check it for all classes